docs/tutorials/clickstream-docker.rst

```diff
@@ -164,17 +166,11 @@ Load the Streaming Data to KSQL
 
 #.  Launch the KSQL CLI in Client Server Mode.
 
-    1. Start the KSQL server.
+    1. Start the CLI and point it to the server.
```

Comment:
Because we aren't actually pointing it to the server should we remove that part of the sentence?